acs5countypoverty.py

```diff
@@ -0,0 +1,57 @@
+from census import Census
+from api_keys import census_key
+import json
+import requests
+import pandas as pd
+
+c = Census(census_key)
+
+#Estimate!!Total!!Population for whom poverty status is determined
+pop_table = 'S1701_C01_001E'
+#Estimate!!Below poverty level!!Population for whom poverty status is determined
+poverty_table = 'S1701_C02_001E'
+
+subject_table = 'https://api.census.gov/data/2018/acs/acs5/subject?'
+get = f'NAME,{pop_table},{poverty_table}'
+#county level request for IL (state 17)
+geo = 'county:*&in=state:17'
+url = f'{subject_table}get={get}&for={geo}&key={census_key}'
+response = requests.get(url)
```

Comment:
💯  Yeah I like using the API instead of IPUMS whenever possible, it makes it more repeatable and less manual